From: Vittorio Gambaletta <openwrt@vittgam.net>
Date: Fri, 01 Jan 2016 00:00:00 +0100
Subject: [PATCH 1/3] Documentation: DT: net: mediatek: Fix documentation for the rt3050 switch driver.

The prefix used in the driver is now "mediatek" instead of "ralink".

Signed-off-by: Vittorio Gambaletta <openwrt@vittgam.net>
---

--- a/Documentation/devicetree/bindings/net/ralink,rt3050-esw.txt
+++ b/Documentation/devicetree/bindings/net/ralink,rt3050-esw.txt
@@ -14,9 +14,9 @@ Required properties:
 - reset-names: Should contain the reset names "esw"
 
 Optional properties:
-- ralink,portmap: can be used to choose if the default switch setup is
+- mediatek,portmap: can be used to choose if the default switch setup is
   llllw or wllll
-- ralink,led_polarity: override the active high/low settings of the leds
+- mediatek,led_polarity: override the active high/low settings of the leds
 
 Example:
 
